tools/CMakeLists.txt

```diff
@@ -19,9 +19,6 @@ endif()
 
```

Comment:
I'm not sure why this patch is here. Can you please rebase on the latest main commit?